### PR TITLE
VEN-85 | Tooltip's Position

### PR DESCRIPTION
**Pull request by Nizar-Rahme**

### CODE REVIEW

File: src/utils/berths.ts [hunk 1]
```diff
@@ -86,3 +86,5 @@ export const isBerthSelected = (
   selectedBerths: Berths,
   berth: BerthType | WinterStorageType
 ): boolean => !!selectedBerths.find(selectedBerth => selectedBerth.id === berth.id);
+
```
Comment (midori0507) [inline src/utils/berths.ts]: Can you put a comment in here explain about the function ?

Comment (midori0507) [reply src/utils/berths.ts]: `docblockr` gonna be amazing

File: apollo.config.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+require('dotenv').config({ path: '.env.development.local' });
```
Comment (midori0507) [inline apollo.config.js]: We should use a pattern in here ? for example `.env.*` ? Or you only need apollo on local?

Comment (Nizar-Rahme) [reply apollo.config.js]: I think it's only needed locally unless we want to generate TS types for queries whenever we invoke Travis for example.
What do you think?

Comment (midori0507) [reply apollo.config.js]: I dont have any plan for it so i guess if you think local is fine then its fine

**Pull request merged**